bxin/ptg2hex/hex_diagnostics.ipynb

```diff
@@ -0,0 +1,607 @@
+    "cmd_radec = cmd_elaz.transform_to(ICRS)\n",
+    "# Calculating the other parameters     \n",
+    "rot_pa = rot_tel\n",
+    "#await rot.cmd_trackStart.start(timeout=30.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#The pointing component is commanding the mount directly\n",
+    "ack = await ptg.cmd_raDecTarget.set_start(\n",
+    "    targetName=target_name,\n",
+    "    frame=ATPtg.CoordFrame.ICRS,\n",
+    "    epoch=2000,  # should be ignored: no parallax or proper motion\n",
+    "    equinox=2000,  # should be ignored for ICRS\n",
+    "    ra=cmd_radec.ra.hour,\n",
+    "    declination=cmd_radec.dec.deg,\n",
+    "    parallax=0,\n",
+    "    pmRA=0,\n",
+    "    pmDec=0,\n",
+    "    rv=0,\n",
+    "    dRA=0,\n",
+    "    dDec=0,\n",
+    "    rotPA=rot_pa.deg-180,\n",
+    "    rotFrame=ATPtg.RotFrame.FIXED,\n",
+    "    rotMode=ATPtg.RotMode.FIELD,\n",
+    "    timeout=10\n",
+    ")\n",
+    "\n",
+    "print(\"Waiting 30s\")\n",
+    "await asyncio.sleep(30.)\n",
+    "print(\"System Ready\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mountStatus = await mount.evt_axesInPosition.aget(timeout=5.)\n",
+    "rotStatus = await rot.evt_inPosition.aget(timeout=5.)\n",
+    "print('Are we tracking?', mountStatus.elevation , mountStatus.azimuth , rotStatus.inPosition)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await ptg.cmd_stopTracking.set_start(timeout=5.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### check angle and LUT after the slew"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mountAngle = await mount.tel_elevation.aget(timeout=10.)\n",
+    "print(\"mount elevation angle\", mountAngle.angleActual)\n",
+    "elev = mountAngle.angleActual"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pos = await hex.tel_application.aget(timeout=10.)\n",
+    "print('hex position')\n",
+    "for i in range(6):\n",
+    "    print('%10.2f '%pos.position[i], end= ' ')\n",
+    "print('\\n')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "posC = await hex.evt_compensatedPosition.aget(timeout=10.)\n",
+    "print('Compensated position')\n",
+    "for i in 'xyzuvw':\n",
+    "    print('%10.2f '%getattr(posC, i), end= ' ')\n",
+    "print('\\n')\n",
+    "posU = await hex.evt_uncompensatedPosition.aget(timeout=10.)\n",
+    "print('Uncompensated position')\n",
+    "for i in 'xyzuvw':\n",
+    "    print('%10.2f '%getattr(posU, i), end= ' ')\n",
```

Comment:
Another case where an f string and comprehension might clarify.

Also another good case for a function, since you can print compensated and uncompensated position with the same code.